homeassistant/components/nexia/config_flow.py

```diff
@@ -88,6 +88,9 @@ async def async_step_user(self, user_input=None):
 
     async def async_step_import(self, user_input):
         """Handle import."""
+        for entry in self._async_current_entries():
+            if entry.data[CONF_USERNAME] == user_input[CONF_USERNAME]:
+                return self.async_abort(reason="already_configured")
```

Comment:
Don't you want this gate in the `user` step?  That way it's covered by `import` and `user`.

Reply:
It is, on line 82 there is `self._abort_if_unique_id_configured()`, which should be the same unique ID for old + new config entry.

Reply:
btw, I do think that we might actually want to drop it from line 82, in case someone has changed their auth and needs to re-authenticate. Now that's not possible.